index.js

```diff
@@ -85,183 +24,245 @@ const defaultNpmScripts = [
   'stop',
   'test',
   'uninstall',
-  'version'
-];
+  'version',
+]
+
+const sortScripts = scripts => {
+  const prefixableScripts = defaultNpmScripts
+  Object.keys(scripts).forEach(script => {
+    const prefixOmitted = script.replace(prefixedScriptRegex, '$2')
+    if (scripts[prefixOmitted] && !prefixableScripts.includes(prefixOmitted)) {
+      prefixableScripts.push(prefixOmitted)
+    }
+  })
+  return sortObjectKeys(compareScriptKeys(toSortKey(prefixableScripts)))(
+    scripts,
+  )
+}
+
+// field.key{string}: field name
+// field.over{function}: sort field subKey
+const fields = [
+  { key: 'name' },
+  { key: 'version' },
+  { key: 'private' },
+  { key: 'description' },
+  { key: 'keywords', over: uniq },
+  { key: 'homepage' },
+  { key: 'bugs', over: sortObjectBy(['url', 'email']) },
+  { key: 'repository', over: sortURLObject },
+  { key: 'funding', over: sortURLObject },
+  { key: 'license', over: sortURLObject },
+  { key: 'author', over: sortAuthorObject },
+  { key: 'files' },
```

Comment:
I think this was sorted before, as array, maybe we can also unique it

Reply:
sorry, it's not sorted before,  in this case

```js
  "files": [
    "index.js",
    "index.d.ts"
  ],
```

I guess , it doesn't make sense to put `index.d.ts` above `index.js`, wo we should keep the order as it is.

add uniq in https://github.com/keithamus/sort-package-json/pull/90/commits/e10cf9ad3389d13e27a0e617419718db6a7ae78a